docs/docs/Agents/agent-tool-calling-agent-component.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Create a problem solving agent
+sidebar_position: 2
+slug: /agents-tool-calling-agent-component
+---
+
+Developing **agents** in Langchain is complex.
+
+The `AgentComponent` is a tool for easily creating an AI agent capable of reasoning through tasks using tools you provide.
+
+The component contains all of the elements you'll need for creating an Agent. Instead of managing LLM models and providers, pick your model and enter your API key. Instead of connecting a **Prompt** component, enter instructions in the component's **Agent Instruction** fields.
+
+<img src="/img/tool-calling-agent-component.png" alt="Prompt component" style={{display: 'block', margin: 'auto', width: 300}} />
+
+Learn how to build a flow starting with the **Tool calling agent** component, and see how it can help you solve problems.
+
+## Prerequisites
+
+- [Langflow installed and running](/getting-started-installation)
+- [OpenAI API key created](https://platform.openai.com/)
+- [Search API key created](https://www.searchapi.io/)
+
+## Create a problem-solving agent
```

Comment:
The title of this page is also "Create a problem-solving agent". Avoid duplicate titles. Can you append either this heading or the main title to create a distinction?